libraries/AP_HAL_Linux/RCInput_ZYNQ.h

```diff
@@ -15,8 +15,13 @@ class RCInput_ZYNQ : public RCInput {
     void _timer_tick(void);
 
 private:
+    #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_OCPOC_ZYNQ
```

Comment:
Indentation of preprocessor directives should be independent of code, so here it should have 0 indentation - just like you did in RCOutput_ZYNQ.h